Source/WebKit/WebKitSwift/TextExtraction/WKWebView+TextExtraction.swift

```diff
@@ -21,7 +21,7 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
 // THE POSSIBILITY OF SUCH DAMAGE.
 
-#if canImport(UIIntelligenceSupport)
+#if HAVE_UIINTELLIGENCESUPPORT_FRAMEWORK && canImport(UIIntelligenceSupport)
```

Comment:
Seems safe to remove the canImport? 
```suggestion
#if HAVE_UIINTELLIGENCESUPPORT_FRAMEWORK
```

Reply:
This one was for sure causing a build error in EWS, I started without it then had to add it back unfortunately. I considered moving the check to the PlatformHave flag, but couldn't find an equivalent

Reply:
I think it's a similar situation to HAVE_WRITING_TOOLS, where the framework is private and we don't have an SPI module available for public use. I think it would make more sense to do what HAVE_WRITING_TOOLS does, and add a `__has_include(<UIIntelligenceSupport/UIIntelligenceSupport.h>)` clause here: https://github.com/WebKit/WebKit/blob/12e840378db6d373821660b7317a98170dad399e/Source/WTF/wtf/PlatformHave.h#L1790-L1797